### DISCUSSION TITLE
queryString property empty on mount

### DISCUSSION THREAD

**Discussion by michele-grifa** [livewire/livewire#4474] [7. Bugs (archived)]
Hi, i have the following component
```php
<?php

namespace App\Http\Livewire\GestioneOf;
use Livewire\Component;

class ListaRigheOc extends Component
{
    public $queryDocumento;

    protected $queryString = [
        'queryDocumento' => ['except' => '', 'as' => 'qdc']
    ];

    public function render()
    {
        return view('livewire.gestione-of.lista-righe-oc');
    }
}

```

```html
<input type="text" class="form-control" placeholder="Documento" aria-label="Documento" wire:model.debounce.750ms="queryDocumento">
```

When i insert a space or a slash in the input, and reload the page, the `queryDocumento` property is empty.
The same component works as expected on 2.7.2

**PHP Version:** 8.0
**Laravel Version:** 8.77.1
**Livewire Version:** 2.8.2

**Comment by joshhanley** [comment]
@michele-grifa hmm I'm not sure I follow, can you confirm if the queryString updates before you reload?

Any chance of doing a screencapture? I use https://gifcap.dev/

**Reply by michele-grifa** [reply]
I double checked everything and the queryString after the reload is correctly updated, but the input with the `wire:model.debounce.750ms` is empty. The problem occurs only when i use the queryString aliases introduced in 2.8.0 release.

![Recording 2022-01-10 at 09 08 44](https://user-images.githubusercontent.com/33720244/148734841-4aa7f838-26bf-45d7-9d8c-9739a069a952.gif)

**Reply by amiranagram** [reply]
Does #4362 fix it for you @michele-grifa? You can try with `dev-master` since it isn't released yet.

**Reply by michele-grifa** [reply]
> Does #4362 fix it for you @michele-grifa? You can try with `dev-master` since it isn't released yet.

How can i test it?

**Reply by amiranagram** [reply]
Replace livewire version with `dev-master` in composer.json and do a `composer update livewire/livewire`.

**Reply by michele-grifa** [reply]
With that PR the problem is solved. Thanks.

**Reply by amiranagram** [reply]
You're welcome. I'm guilty of that bug 🤷‍♂️, sorry.

**Reply by joshhanley** [reply]
@michele-grifa thanks for testing and confirming, that fix will be in the next release 🙂 

Thanks @amiranagram for your support here!
